packages/appium/docs/en/commands/base-driver.md

```diff
@@ -217,6 +217,8 @@ A possibly empty list of elements inside the shadow root matching the selector
 
 Get the log for a given log type.
 
+**`Deprecated`**
```

Comment:
Can these warnings be changed to the MkDocs format and moved right below the command heading?
```
!!! warning "Deprecated"

    Please use the `<endpoint>` endpoint instead
```
[This format is already used in the XCUITest driver.](https://github.com/appium/appium-xcuitest-driver/blob/master/docs/reference/commands.md?plain=1#L332)

Reply:
Ok nevermind, this is already being done in the other PR.